### PR TITLE
Document new features

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,10 +27,17 @@ Originally developed by [Sean Sackowitz](https://github.com/seansackowitz).
 Go to **System Console > Plugins > Jitsi** and set the following values:
 
 1. **Enable Plugin**: ``true``
-2. **Jitsi URL**: The URL for your on-premise Jitsi server. If you set the Jitsi Server URL to https://meet.jit.si it uses the public server provided by Jitsi.
-3. (Optional) If your Jitsi server uses JSON Web Tokens (JWT) for authentication, set **Use JWT Authentication for Jitsi** to true, and specify the **App ID** and **App Secret** used for JWT authentication.
-4. (Optional) **Meeting Link Expiry Time** in minutes. Defaults to 30 minutes.
-5. **Jitsi Meeting Names** for auto-generated Jitsi meetings. Defaults to using random English words in title case, but you can also use a UUID as the meeting link, or the team and channel name where the Jitsi meeting is created.
+2. **Jitsi Server URL**: The URL for your on-premise Jitsi server. If you set the Jitsi Server URL to https://meet.jit.si it uses the public server provided by Jitsi.
```
Comment (jasonblais) [inline README.md:30]: "The URL for your on-premise Jitsi server" -- is this true? Because they could also use the public service meet.jit.si?

Comment (jespino) [reply README.md:30]: There are not necesarilly on-premise, actually, you can use the official one, third party ones, or something on-premise. For example, you can use one of this list here: https://github.com/jitsi/jitsi-meet/wiki/Jitsi-Meet-Instances

Probably you should ask before use one of that, but well... there are a lot of jitsi server running and not necesarilly on-prem.

Comment (jasonblais) [reply README.md:30]: @jespino That's good to know. What if we then just remove `on-premise` and say `The URL for your Jitsi server. If you set the Jitsi Server URL to https://meet.jit.si it uses the public server provided by Jitsi.`

I can remove `on-premise` from the System Console setting help text as well.

Comment (jespino) [reply README.md:30]: Yes, I think that is better. Thanks :) and yes, in both places, please :)

**Comment by jasonblais** [comment]
@jespino Updated in both by removing `on-premise`

**Pull request merged**